common/model-views.cpp

```diff
@@ -4973,7 +5021,23 @@ namespace rs2
                 {
                     if( auto nm = notification_model_protected.lock() )
                     {
-                        check_for_bundled_fw_update( ctx, nm );
+                        fw_bundled_update_available = check_for_bundled_fw_update( ctx, nm , activated_by_user);
+                    }
+                }
+
+                // When no updates available (on-line + bundled), add a notification to indicate "all up to date"
+                if( activated_by_user && ! fail_access_db && ! sw_online_update_available
+                    && ! fw_online_update_available && ! fw_bundled_update_available )
+                {
+                    auto n = std::make_shared< sw_update_up_to_date_model >();
+                    auto name = get_device_name(dev);
+                    n->delay_id = "no_updates_alert." + name.second;
+                    n->enable_complex_dismiss = true;  // allow advanced dismiss menu
```

Comment:
Do we really want to allow this to be dismissed?

Reply:
Not more than just now (Copy paste mistake), complex dismissed removed